src/RuleSet/Sets/PERCS3x0Set.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ *
+ * PER Coding Style v3.0.
+ *
+ * @see https://github.com/php-fig/per-coding-style/blob/3.0.0/spec.md
```

Comment:
sounds goodish, but need some time to process it and starting my holidays soon. thanks for your upcoming patience !